novemberfive_logging/_logger.py

```diff
@@ -0,0 +1,108 @@
+import contextlib
+import logging
+import os
+import sys
+import uuid
+
+import structlog
+from structlog import contextvars
+
+from novemberfive_logging import _processors
+from novemberfive_logging import _fields
+
+
+def _get_log_level():
+    if os.getenv("LOG_LEVEL") is not None:
+        log_level = os.environ['LOG_LEVEL']
+    elif os.getenv("ENV") == 'dev':
+        log_level = logging.DEBUG
+    else:
+        log_level = logging.INFO
+
+    return log_level
+
+
+def _get_renderer():
+    if "ENV" not in os.environ:
```

Comment:
ik zou hier nog "or os.getenv("ENV") == 'dev'" aan toevoegen?

Reply:
Dat kan, heb ik over nagedacht. Willen we pretty logging als env DEV is? Die logging is specifiek geformat voor de terminal, ik denk dat we in de echte develop omgeving best JSON formaat aanhouden.

Reply:
Als ENV='dev' betekent die lokale omgeving, dus deze change neem ik nog mee.